.buildkite/packer_cache.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+ROOT_DIR=$(cd "$(dirname "$0")/.." && pwd)
+CI_DIR="${ROOT_DIR}/.ci"
+
+if [ "$(uname -m)" = "arm64" ] || [ "$(uname -m)" = "aarch64" ]; then
+  ## On ARM we use a different properties file for setting java home
+  ## Also, we don't bother attempting to resolve dependencies for the 6.8 branch
+  source "$CI_DIR/java-versions-aarch64.properties"
+  export JAVA16_HOME="$HOME/.java/jdk16"
+else
+  source "$CI_DIR/java-versions.properties"
```

Comment:
I think we need to do this `source` bit in the loop below since this is going to be different for each branch.